tools/openocd/modm/atmel_same70_xplained.cfg

```diff
@@ -0,0 +1,14 @@
+#
+# Atmel SAMV71 Xplained Ultra evaluation kit.
+# http://www.atmel.com/tools/ATSAMV71-XULT.aspx
+#
+# To connect using the EDBG chip on the dev kit over USB, you will
+# first need to source [find interface/cmsis-dap.cfg]
+# however, since this board also has a SWD+ETM connector, we don't
+# automatically source that file here.
```

Comment:
comment seems to be out of sync with the next line ;-P

Reply:
Yes, I'll fixed. thanks

Reply:
In fact I saw that I made a mistake. Now I updated with the right configuration, same as openocd.